src/redux/books/books.js

```diff
@@ -1,24 +1,55 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
-
+const GET_BOOKS = 'bookStore/books/GET_BOOKS';
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/QY9p53fQNfR1kQ69d82L/books';
 const initialState = [];
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
-export const removeBook = (payload) => ({
-  type: REMOVE_BOOK,
-  payload,
-});
+export const removeBook = (payload) => async (dispatch) => {
+  const { data } = await axios.delete(
+    `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/QY9p53fQNfR1kQ69d82L/books/${payload}`,
+  );
```

Comment:
- [OPTIONAL] It will be better to take benefit of the API variable and use it here.
```
const { data } = await axios.delete(`${url}/${payload}`); 
```

Reply:
Thank you for the feedback